docs/databases/connecting.md

```diff
@@ -126,8 +126,6 @@ During the scan, Metabase also takes a sample of each table to look for URLs, JS
 1. Go to **Admin settings** > **Databases** > your database.
 2. Click on **Sync database schema now**.
 
-![Database Manual Sync](./images/DatabaseManualSync.png)
-
```

Comment:
It's part of our [style guide](https://www.notion.so/metabase/Style-tips-b1e79eae0efb4e48b890d8c6555cb174#8aed5381210843eba11098af65d08066)!

> Where possible, avoid images and gifs (or even remove them)
Your future selves will not thank you, because the won't even know about the boatloads of time you've saved them by not giving them more assets they'll need to update when the product inevitably changes.
Gut check: if you think people can probably figure out instructions without an image, it doesn't need an image.